girder/api/v1/user.py

```diff
@@ -235,11 +235,11 @@ def updateUser(self, user, firstName, lastName, email, admin, status):
         user['email'] = email
 
         # Only admins can change admin state
-        if admin is True:
+        if admin is not None:
             if self.getCurrentUser()['admin']:
                 user['admin'] = admin
-            elif not user['admin']:
-                raise AccessException('Only admins may enable admin state.')
+            elif user['admin'] is not admin:
+                raise AccessException('Only admins may change admin status.')
```

Comment:
When a non-admin specifies the `admin` parameter, I suggest raising this exception regardless of the existing value. What do you think?

Reply:
For backward compatibility, I'd prefer not to change that behavior at this time. The `admin` field of user documents is exposed to everyone, so if someone attempts to naively save the user document with all of its fields that were originally sent down to them as params, it would fail after the proposed change.

Reply:
That makes sense, thanks.